### PR TITLE
Update exception handling

### PR DESCRIPTION
**Pull request by rctauber**
See #304

### CODE REVIEW

File: robot-command/src/main/java/org/obolibrary/robot/CommandLineHelper.java [hunk 1]
```diff
@@ -613,7 +613,7 @@ public static IRI maybeCreateIRI(IOHelper ioHelper, String term, String field) {
    */
   public static Set<IRI> getTerms(IOHelper ioHelper, CommandLine line, String singles, String paths)
       throws IllegalArgumentException, IOException {
-    Set<String> termStrings = new HashSet<String>();
+    Set<String> termStrings = new HashSet<>();
```
Comment (jamesaoverton) [inline robot-command/src/main/java/org/obolibrary/robot/CommandLineHelper.java]: Why change `HashSet<String>` to `HashSet<>` (and a few others)?

Comment (rctauber) [reply robot-command/src/main/java/org/obolibrary/robot/CommandLineHelper.java]: It's less verbose - https://docs.oracle.com/javase/8/docs/technotes/guides/language/type-inference-generic-instance-creation.html

I can change it back if you prefer, it's the same in terms of performance, just makes the code a little easier to read.

Comment (jamesaoverton) [reply robot-command/src/main/java/org/obolibrary/robot/CommandLineHelper.java]: Unless we're going to change it across the whole code base, I'd prefer to stick with the more explicit style for consistency.

Comment (jamesaoverton) [reply robot-command/src/main/java/org/obolibrary/robot/CommandLineHelper.java]: I guess it's not consistent. Just leave it, it doesn't matter.

**Comment by jamesaoverton** [comment]
Closing without merge in favour of #307.

**Pull request closed without merging**